images/scripts/arch.install

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+set -e
+
+/var/lib/testvm/arch.install "$@"
```

Comment:
Arch does not have multiple releases, and there is no parameterization here. Please just move arch.install from scripts/lib/ to scripts and avoid the indirection.